src/maps/workerCollection.ts

```diff
@@ -4,23 +4,26 @@ import {
   WorkerCollectionItemResponse
 } from '@/models'
 import { MapFunction } from '@/services/Mapper'
-import { SchemaValues, WorkerBaseJobTemplate, SchemaProperty } from '@/types/schemas'
+import {
+  SchemaValues,
+  WorkerBaseJobTemplate,
+  SchemaProperty
+} from '@/types/schemas'
 
 export const mapPrefectWorkerCollectionResponseToWorkerCollectionItemArray: MapFunction<
 PrefectWorkerCollectionResponse,
 WorkerCollectionItem[]
 > = function(source) {
-  return Object.values(source).reduce<WorkerCollectionItemResponse[]>(
-    (acc, package_data) => [...acc, ...Object.values(package_data)],
-    [],
-  ).map((worker_data) => ({
-    defaultBaseJobConfiguration: worker_data.default_base_job_configuration,
-    description: worker_data.description,
-    documentationUrl: worker_data.documentation_url,
-    installCommand: worker_data.install_command,
-    logoUrl: worker_data.logo_url,
-    type: worker_data.type,
-  }))
+  return Object.values(source)
+    .reduce<WorkerCollectionItemResponse[]>((acc, package_data) => [...acc, ...Object.values(package_data)], [])
```

Comment:
Unfortunately, `.flat()` doesn't work in this case because we're dealing with nested objects, but I was able to use `.flatMap()` to clean the logic up a bit.

Reply:
Ah that makes, sense. I was struggling to pick apart that reduce. Much cleaner ty